cirq-superstaq/cirq_superstaq/service_test.py

```diff
@@ -249,40 +249,13 @@ def test_service_get_balance() -> None:
     assert service.get_balance(pretty_output=False) == 12345.6789
 
 
-def test_service_get_targets() -> None:
+@mock.patch(
+    "general_superstaq.superstaq_client._SuperstaqClient.get_request",
+    return_value={"superstaq_targets": gss.testing.TARGET_LIST},
+)
+def test_service_get_targets(mock_get_request: mock.MagicMock) -> None:
```

Comment:
(assuming prior) this should also be moved to `gss.service_test`

Reply:
Done